pkg/proto/nbcontract/v1/gpuconfig.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package nbcontract.v1;
+
+import "pkg/proto/nbcontract/v1/enum.proto";
+
+message GpuConfig {
+  // nvidia_state represents the Nvidia GPU state
+  optional FeatureState nvidia_state = 1;
+
+  // FeatureState is_mig_node = 2; removed since it only depends on gpu_instance_profile
+
+  //config_gpu_driver represents if bootstrap process should install and configure the GPU driver when necessary. Configuration includes appropriate set up of components like the fabric manager where applicable.
+  optional FeatureState config_gpu_driver = 2;
+
+  //gpu_device_plugin represents if special config is needed for MIG GPUs that use GPU dedicated VHDs and enable the device plugin (for all GPU dedicated VHDs)
+  optional FeatureState gpu_device_plugin = 3; 
+
+  // FeatureState fabric_manager_status = 5;  removed. consider just extracting vm_size since both GPUNeedsFabricManager and getGPUDriverVersion are just depending ont vm_size.
```

Comment:
```suggestion
  // FeatureState fabric_manager_status = 5;  removed. consider just extracting vm_size since both GPUNeedsFabricManager and getGPUDriverVersion depend only on the vm_size.
```

Reply:
Thanks for catching this. We will eventually remove all the comments here. For now they are here for us to track where they go more easily.